### PR TITLE
fix: revert TradingWS to tiles, blotter and RA

### PR DESCRIPTION
**Pull request by algreasley**

### CODE REVIEW

File: src/workspace/config/snapshot.json [hunk 1]
```diff
@@ -1,183 +1,110 @@
 {
   "snapshotDetails": {
-    "timestamp": "2023-06-20T12:17:19.360Z",
-    "runtimeVersion": "29.108.73.14",
+    "timestamp": "2022-01-10T11:26:57.741Z",
+    "runtimeVersion": "23.96.67.7",
```
Comment (SHolleworth) [inline src/workspace/config/snapshot.json:4]: should this have been reverted all the way back to 23?

File: src/workspace/config/snapshot.json [hunk 2]
```diff
@@ -247,121 +175,45 @@
...
+        }
+      },
       "saveWindowState": false,
       "shadow": false,
       "showTaskbarIcon": true,
       "showBackgroundImages": false,
       "smallWindow": false,
       "spellCheck": false,
-      "state": "normal",
+      "state": "maximized",
```
Comment (SHolleworth) [inline src/workspace/config/snapshot.json:205]: do we definitely want it to maximise when it's launched?

File: src/workspace/config/snapshot.json (hunk 1 above)
Comment (algreasley) [reply src/workspace/config/snapshot.json:4]: it doesn't matter .. that was working before I updated the snapshot

File: src/workspace/config/snapshot.json (hunk 2 above)
Comment (algreasley) [reply src/workspace/config/snapshot.json:205]: again, just restoring previous

**Review by SHolleworth — APPROVED** [review]

**Pull request merged**